lib/safemode/core_jails.rb

```diff
@@ -153,137 +211,309 @@ def core_jail_class_methods(klass)
+    compare_by_identity?
+    count
+    cycle
+    default
+    default=
+    default_proc
+    default_proc=
     delete
     delete_if
+    detect
+    dig
+    drop
+    drop_while
     each
+    each_cons
+    each_entry
     each_key
     each_pair
+    each_slice
     each_value
+    each_with_index
+    each_with_object
     empty?
+    entries
     fetch
+    fetch_values
+    find
+    find_all
+    find_index
+    first
+    flat_map
+    flatten
+    grep
+    grep_v
+    group_by
     has_key?
     has_value?
     include?
     index
+    inject
     invert
+    keep_if
+    key
     key?
     keys
+    lazy
     length
+    map
+    max
+    max_by
     member?
     merge
     merge!
-    present?
-    rec_merge!
+    min
+    min_by
+    minmax
+    minmax_by
+    none?
+    one?
+    partition
+    rassoc
+    reduce
     rehash
     reject
     reject!
+    replace
+    reverse_each
     select
+    select!
     shift
     size
+    slice
+    slice_after
+    slice_before
+    slice_when
     sort
+    sort_by
     store
+    sum
+    take
+    take_while
+    to_h
+    to_hash
+    to_proc
+    to_set
+    transform_keys
+    transform_keys!
+    transform_values
+    transform_values!
+    uniq
     update
     value?
     values
     values_at
+    zip
   ),
   'Range' => %w(
+    all?
     any?
     begin
-    blank?
+    bsearch
+    chunk
+    chunk_while
+    collect
+    collect_concat
+    count
+    cover?
+    cycle
+    detect
+    drop
+    drop_while
     each
+    each_cons
+    each_entry
+    each_slice
+    each_with_index
+    each_with_object
     end
+    entries
     exclude_end?
+    find
+    find_all
+    find_index
     first
-    hash
+    flat_map
+    grep
+    grep_v
+    group_by
     include?
-    include_without_range?
+    inject
     last
+    lazy
+    map
+    max
+    max_by
     member?
-    present?
+    min
+    min_by
+    minmax
+    minmax_by
+    none?
+    one?
+    partition
+    reduce
+    reject
+    reverse_each
+    select
+    size
+    slice_after
+    slice_before
+    slice_when
+    sort
+    sort_by
     step
+    sum
+    take
+    take_while
+    to_h
+    to_set
+    uniq
+    zip
   ),
   'String' => %w(
-    blank?
+    ascii_only?
+    b
+    between?
+    bytes
+    bytesize
+    byteslice
     capitalize
     capitalize!
     casecmp
+    casecmp?
     center
+    chars
+    chomp
+    chomp!
+    chop
+    chop!
+    chr
+    clamp
+    clear
+    codepoints
     concat
     count
     crypt
     delete
     delete!
+    delete_prefix
+    delete_prefix!
+    delete_suffix
+    delete_suffix!
     downcase
     downcase!
     dump
     each_byte
+    each_char
+    each_codepoint
+    each_grapheme_cluster
     each_line
     empty?
+    encode
+    encode!
+    encoding
     end_with?
+    ext
     force_encoding
+    getbyte
+    grapheme_clusters
+    grep
     gsub
     gsub!
-    hash
     hex
-    chomp
-    chomp!
-    chop
-    chop!
```

Comment:
Oh this is just a problem with sort - I sorted using Czech locale and in our language "ch" is actually one characters, it's sorted differently. See above.